data/english.cfg

```diff
@@ -21,8 +21,10 @@ Twilight: +25% Damage"
     #ranges
     range_melee= _ "melee"
     range_ranged= _ "ranged"
+    range_rrr=_ "rrr"
 
     #damage types
+    type_ttt=_ "ttt"
```

Comment:
I added these to test units that don't have range or damage type graphics. I'll revert them and rebase before merge.

Reply:
ok